helm/app-admission-controller/templates/configmap.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Why have this extra configmap instead of directly doing it in the deployment? I am concerned what happens if the configmap is changed. I don't think it will auto reload.

Reply:
The reason was aesthetics 😄 I didn't want to change how we get the config, but at the same time `kingpin` gives limited possibilities for providing configuration, I mean it doesn't support more sophisticated config files. And since the number of flags is decent I think, I decided to try to use a file for keeping them, hoping it would look better. But I can change that of course

Reply:
done